python/paddle/amp/amp_lists.py

```diff
@@ -118,7 +121,7 @@ def white_list():
     return white_list
 
 
-def black_list():
+def black_list() -> dict[str, dict[str, Any]]:
```

Comment:
```suggestion
def black_list() -> dict[str, dict[str, set[str]]]:
```

应该也还是 `set[str]` 的样子 ～

另外，@SigureMo 这里有必要用 `TypedDict` 吗?

Reply:
不用了吧，这又不是公开 API，这里的输入输出的「协议」让开发的人自己把握就好了，改成 TypedDict 他们反而维护不起来了